modelexpress_client/python/modelexpress/envs.py

```
@@ -0,0 +1,211 @@
+    "MX_RDMA_NIC_PIN": lambda: os.environ.get("MX_RDMA_NIC_PIN", "").strip(),
+    "MX_RDMA_NIC_PIN_MIN_RATE_GBPS": lambda: os.environ.get("MX_RDMA_NIC_PIN_MIN_RATE_GBPS"),
+    # ── GPUDirect Storage ──────────────────────────────────────────────────
+    "MX_GDS_MAX_CHUNK_KB": lambda: os.environ.get("MX_GDS_MAX_CHUNK_KB"),
+    "MX_GDS_THREADS": lambda: _env_int("MX_GDS_THREADS", 8),
+    "MX_GDS_TIMEOUT": lambda: _env_float("MX_GDS_TIMEOUT", 120.0),
+    # ── Model streamer ─────────────────────────────────────────────────────
+    "MX_MS_DISTRIBUTED": lambda: os.environ.get("MX_MS_DISTRIBUTED", "0").lower() in ("1", "true"),
+    # ── TRT-LLM live transfer ──────────────────────────────────────────────
+    "MX_SOURCE_QUERY_TIMEOUT": lambda: _env_int("MX_SOURCE_QUERY_TIMEOUT", 3600),
+    "MX_TRANSFER_TIMEOUT": lambda: _env_int("MX_TRANSFER_TIMEOUT", 900),
+    "MX_TRANSFER_LOG_DIR": lambda: os.environ.get("MX_TRANSFER_LOG_DIR", "/tmp/mx_logs"),
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/mx_logs"

(S108)